test/src/main/java/org/apache/accumulo/harness/AccumuloITBase.java

```diff
@@ -111,19 +111,18 @@ public static File createTestDir(String name) {
     }
 
     // if either value is zero, apply a very long timeout (effectively no timeout)
-    int totalTimeoutSeconds = timeoutFactor * defaultTimeoutSeconds();
-    if (totalTimeoutSeconds == 0) {
+    if (timeoutFactor == 0 || defaultTimeoutDuration().isZero()) {
```

Comment:
If the the `defaultTimeoutDuration()` is 0, it should really be an error. We don't want infinite timeouts by default, only if the user explicitly set it with `-Dtimeout.factor=0`.

Reply:
Addressed in [c03f65e](https://github.com/apache/accumulo/pull/2580/commits/c03f65e720d08abd1341872295d7faaa241d3b93)

Reply:
> We don't want infinite timeouts by default, only if the user explicitly set it with `-Dtimeout.factor=0`.

I don't think the timeout will be infinite by default. I think the only cases where the infinite timeout would be applied (before [c03f65e](https://github.com/apache/accumulo/pull/2580/commits/c03f65e720d08abd1341872295d7faaa241d3b93)) would be if the user explicitly set it with `-Dtimeout.factor=0`, or if `defaultTimeout()` was explicitly overridden to return a duration of zero.

Reply:
> or if `defaultTimeout()` was explicitly overridden to return a duration of zero.

Right. That's the case I'm talking about. This shouldn't happen. This should be an error. The only time it should be infinite is if the user says so explicitly with `-Dtimeout.factor=0`. None of our tests should ever have that as a their overridden default.